### PR TITLE
Build: Release PR Release candidate to Master PR for version 1.6.14

### PR DESCRIPTION
**Pull request by mahajanleena1995**
### Description
- Final release PR. Please check all the checklist below before merging.
---
### Checklist:
- [x] Check if astra-notice library is updated to latest
- [x] Check if Elementor & Elementor Pro compatibility tags are updated to latest
- [x] Update - Tested Upto - readme.txt
- [x] Update Readme.txt file and then run: `grunt readme`
- [x] Run to update version number  : `grunt version-bump --ver=<version-number>`
- [x] Generate POT file             : `grunt i18n`
- [x] PHPCS - vendor/bin/phpcs
- [x] Add draft a tag GitHub
- [x] Open PR to backport branch to `next-release`
---

### Description
Changelog - 
- Improvement: Compatibility with Elementor version 3.13 and Elementor Pro version 3.13
- Fix: Navigation Menu - Accessibility error corrected for [aria-hidden="true"] attribute.

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by vrundakansara — APPROVED** [review]

**Review by sushmak02 — APPROVED** [review]

File: inc/js/frontend.js [hunk 1]
```diff
@@ -167,8 +167,16 @@
...
+		var hef_navmenu_toggle = $scope.find( '.hfe-nav-menu__toggle' );
+		hef_navmenu_toggle.attr( 'aria-haspopup', 'true' );
+		hef_navmenu_toggle.attr( 'aria-expanded', 'false' );
+
+		if ( window.matchMedia( "( max-width: 1024px )" ).matches && $( '.elementor-element-' + id ).hasClass('hfe-nav-menu__breakpoint-tablet') ) {
+			hef_navmenu_toggle.find('i').attr('aria-hidden', 'false');
+		}
+		if ( window.matchMedia( "( max-width: 768px )" ).matches && $( '.elementor-element-' + id ).hasClass('hfe-nav-menu__breakpoint-mobile') ) {
+			hef_navmenu_toggle.find('i').attr('aria-hidden', 'false');
+		}
```
Comment (patilvikasj) [inline inc/js/frontend.js:179]: @mahajanleena1995 We can combine these 2 if conditions. I guess the only difference is of max-width value.

**Review by patilvikasj — CHANGES_REQUESTED** [review]
Please check comment

File: inc/js/frontend.js (hunk 1 above)
Comment (mahajanleena1995) [reply inc/js/frontend.js:179]: @patilvikasj `hasClass` condition also have different values, should I still combine both using OR

Comment (patilvikasj) [reply inc/js/frontend.js:179]: Got it. No need to combine them.

File: inc/widgets-manager/widgets/class-navigation-menu.php [hunk 2]
```diff
@@ -2017,7 +2017,8 @@ protected function render() {
 
 			?>
 			<div <?php echo $this->get_render_attribute_string( 'hfe-main-menu' ); ?>>
-				<div class="hfe-nav-menu__toggle elementor-clickable">
+				<div role="button" class="hfe-nav-menu__toggle elementor-clickable">
+					<span class="screen-reader-text">Menu</span>
```
Comment (patilvikasj) [inline inc/widgets-manager/widgets/class-navigation-menu.php:2021]: This `Menu` string needs to be translatable.

Comment (mahajanleena1995) [reply inc/widgets-manager/widgets/class-navigation-menu.php:2021]: Updated here - https://github.com/brainstormforce/header-footer-elementor/pull/673

**Review by RajkiranBagal — APPROVED** [review]

**Review by patilvikasj — APPROVED** [review]

**Pull request merged**